apps/components-e2e/src/components/tag-list/tag-list.e2e.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Selector } from 'testcafe';
+
+const tagList = Selector('#tagList');
+
+fixture.only('TagList').page('http://localhost:4200/tag/tag-list');
+
+test("should not display '0 More...'", async (testController: TestController) => {
+  await testController.resizeWindow(700, 500);
+  await testController.expect(tagList.textContent).notContains('0 More...');
```

Comment:
chain the expect to the resize and add a `.wait(300)` between those to wait a little bit after resizing